packages/backend/src/models/execution.ts

```diff
@@ -8,6 +8,7 @@ class Execution extends Base {
   testRun: boolean
   internalId: string
   executionSteps: ExecutionStep[]
+  status: string
```

Comment:
smoll nit:

type ExecutionStatus = 'success' | 'failure' | null

```suggestion
  status: ExecutionStatus
```

Reply:
okk this is good, thanks